drivers/sensor/vl53l0x/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+zephyr_sources_ifdef(CONFIG_VL53L0X vl53l0x.c)
+zephyr_sources_ifdef(CONFIG_VL53L0X vl53l0x_platform.c)
```

Comment:
You should omit the ifdef because this entire directory is already ifdef'ed on CONFIG_VL53L0X.

Ifdef'ing on VL53L0X in here might communicate to the reader that VL53L0X is not always defined when processing this CMakeLists.txt.

So

zephyr_sources(
  vl53l0x.c
  vl53l0x_platform.c
)

Reply:
Thanks Sebastian, good remark !
I have integrated your modification.

Reply:
Are you going to change the CMakeLists.txt?